packages/design-system/src/components/UsaBanner/UsaBanner.jsx

```diff
@@ -0,0 +1,62 @@
+import DotGovIcon from './DotGovIcon';
+import HttpsIcon from './HttpsIcon';
+import LockIcon from './LockIcon';
+import PropTypes from 'prop-types';
+import React from 'react';
+import UsaFlagIcon from './UsaFlagIcon';
+
+const UsaBanner = function (props) {
+  const bannerHeader = props.isBannerOpen
+    ? 'ds-c-usa-banner__header ds-c-usa-banner__header--expanded'
+    : 'ds-c-usa-banner__header';
+
+  return (
+    <div className="ds-c-usa-banner">
+      <header className={bannerHeader}>
+        <UsaFlagIcon />
+        <p className="ds-c-usa-banner__header-text">
+          <span>An official website of the United States government</span>
+          <button
+            onClick={props.onToggleBanner}
+            className="ds-c-usa-banner__button"
+            aria-expanded={props.isBannerOpen}
+            aria-controls="gov-banner"
+          >
+            Here’s how you know
+          </button>
+        </p>
+      </header>
+      <div className="ds-c-usa-banner__content" id="gov-banner" hidden={!props.isBannerOpen}>
+        <div className="ds-u-display--flex ds-u-flex-direction--column ds-u-sm-flex-direction--row ds-u-flex-wrap--nowrap">
```

Comment:
I think these utilities can still be handled by `ds-c-usa-banner__content`

Reply:
I think we need to keep that internal div because the `ds-c-usa-banner__content` is using hidden to show and hide the HTML element and If we set that div to use `display:flex` that will take precedent over the hidden. 
See the note here: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden

Reply:
ok that makes sense. maybe we can add a new class instead? we can call it  content wrapper?